docs/source/whatsnew.rst

```diff
@@ -55,7 +59,7 @@ offers several improvements over the previous lifecycle:
   like to track different types of assets in their own ‘copy’ of the chaincode.
 
 * **Chaincode packages do not need to be identical across channel members**
-  Organizations can customize a chaincode for their own use case, for example
+  Organizations can extend a chaincode for their own use case, for example
   to perform different validations in the interest of their organization.
   As long as the required number of organizations endorse chaincode transactions
   with matching results, the transaction will be validated and committed to the
```

Comment:
Is it worth additionally pointing out here that each organization can run the chaincode in the language of their choice?

Reply:
> Is it worth additionally pointing out here that each organization can run the chaincode in the language of their choice?

I think we agreed / decided not to emphasize this initially

Reply:
no, not something we want to encourage! it's a slippery slope that gets more treacherous as you go down. e.g. json marshaling may be different on each language. so, i'm trying to describe the new 'feature' in a limited way, e.g. as a way to extend a chaincode for additional organization validations.

Reply:
+1. Seems worthwhile to point out this additional flexibility the new chaincode enables.

Reply:
If we don't mention at all, I suspect people will think it is a flaw. So I'm thinking we need to make it explicit, although position it as a way to extend chaincodes in a limited way like adding organization specific validations. I think that aspect is both valuable and feasible for organizations. But not go so far as to recommend completely different chaincodes or different languages. If you still think it should be entirely removed, I can remove it though...